moveit_ros/moveit_servo/config/servo_parameters.yaml

```diff
@@ -239,7 +240,7 @@ servo:
     description: "[Hz] Collision-checking can easily bog down a CPU if done too often. \
                   Collision checking begins slowing down when nearer than a specified distance.",
     validation: {
-      gt_eq<>: 0.0
+      gt_eq<>: 10.0
```

Comment:
I feel like for some applications a min of 10 Hz might be too high, I think greater than zero is ok